com.stansassets.build/Editor/Steps/UnityPlayerBuildStep.cs

```diff
@@ -12,27 +13,27 @@ public class UnityPlayerBuildStep : IBuildStep
 
         public int Priority => m_Priority;
 
-        public bool Execute(BuildContext buildContext)
-        {
-            return BuildProject();
-        }
+        public event Action<ExecuteFinishedArgs> OnExecuteFinished;
+        private event Action onCompleteCallback;
+
 
-        public string GetResultMessage()
+        public void Execute(BuildContext buildContext, Action onComplete)
         {
-            return m_resultMessage;
+             BuildProject();
         }
 
-        private bool BuildProject()
+        private void BuildProject()
         {
-            bool isSuccess = true;
-
             BuildPlayerOptions defaultBuildPlayerOptions = new BuildPlayerOptions();
             BuildPlayerOptions currentBuildPlayerOptions =
                 BuildPlayerWindow.DefaultBuildMethods.GetBuildPlayerOptions(defaultBuildPlayerOptions);
 
+            ExecuteFinishedArgs executeFinishedArgs = new ExecuteFinishedArgs();
```

Comment:
Talke about that in the Nazar's PR

Reply:
We can leave it as is just for the sake for merging this and add tasks so we won't forget to change it